### DISCUSSION TITLE
Logging in with ssh?

### DISCUSSION THREAD

**Discussion by amca01** [mealie-recipes/mealie#2565] [Q&A]
I've got mealie installed on my VPS, and I can access it with 

    http://www.mysite.net:9925

However, I have set up a DNS site 

    mealie.mysite.net

which I'd like to access with https - the certificates being provided by letsencrypt and managed by traefik.  However, I haven't managed it yet.  I can only access mealie via port 9925 as above.  Where can I find some information about setting up encrypted access to mealie?  Thank you!

**Comment by Kuchenpirat** [comment]
Hey, this is not really an issue with mealie as you would usually manage the certs in your reverse-proxy in your case traefik. 

If traefik and mealie are using the same network, traefik should be able to detect the mealie container and route to it. 
Traefik settings for your mealie instanse can be set as labels in your docker compose. 

Down below you can find my old basic traefik config for mealie. It does accept also http requests but reroutes them to https

    labels:
      - "traefik.enable=true"
      # HTTP      
      - "traefik.http.routers.mealie.entrypoints=web"
      - "traefik.http.routers.mealie.rule=Host(`mealie.<your domain here>`)"
      - "traefik.http.middlewares.https-redirect.redirectscheme.scheme=https"
      - "traefik.http.routers.mealie.middlewares=https-redirect"
      # HTTPS
      - "traefik.http.routers.mealie-secure.entrypoints=websecure"
      - "traefik.http.routers.mealie-secure.rule=Host(`mealie.<your domain here>`)"
      - "traefik.http.routers.mealie-secure.tls=true"
      - "traefik.http.routers.mealie-secure.tls.certresolver=myresolver"
